gen/rules/src/main/scala/clue/gen/GraphQLGen.scala

```diff
@@ -129,7 +133,8 @@ class GraphQLGen(config: GraphQLGenConfig)
                             s"Warning parsing document: ${queryResult.toProblems.map(_.toString).toList.mkString("\n")}"
                           )
                         ) >> IO {
-                          val operation = queryResult.toOption.get
+                          // TODO Support multi-operation queries?
```

Comment:
What is a multi-operation query?

Reply:
It's sending multiple queries in the same request.

Here's an example: https://dgraph.io/docs/v21.03/graphql/api/multiples/